ui/app/com/gu/recipeasy/controllers/Application.scala

```diff
@@ -102,6 +102,18 @@ class Application(override val wsClient: WSClient, override val conf: Configurat
     Ok(views.html.recentactivity(userEventDBs))
   }
 
+  def statusDistribution = AuthAction { implicit request =>
+    val distribution: Map[RecipeStatus, Long] = Map(
+      New -> db.countRecipesInGivenStatus(New),
```

Comment:
That being said, I will be happy to make a change, mostly to learn more about Quill.